Orange/widgets/utils/__init__.py

```diff
@@ -32,8 +33,8 @@ def getdeepattr(obj, attr, *arg, **kwarg):
     return deepgetattr(obj, attr, *arg, **kwarg)
 
 
-def to_html(str):
-    return str.replace("<=", "&#8804;").replace(">=", "&#8805;").\
+def to_html(str_):
+    return str_.replace("<=", "&#8804;").replace(">=", "&#8805;").\
         replace("<", "&#60;").replace(">", "&#62;").replace("=\\=", "&#8800;")
```

Comment:
What about `return escape(str_, entities={"<=": "&#8804;", ">=": "&#8805;", "=\\=": "&#8800;"})`?

By the way, for long sequences of replacements I sometimes prefer

```python
reduce(lambda s, t:s.replace(*t), (
    ("<=", "&#8804;"), 
    (">=", "&#8805;"),
    ("<", "&#60;"),
    (">", "&#62;"),
    ("=\\=", "&#8800;")
    )), s)
```

But this is a matter of taste, and `escape` fits better here anyway.

Reply:
Same as above..